docs/content/grafana/add_grafana_dashboard.md

```diff
@@ -41,4 +41,13 @@ $ cd grafana/overlays/moc/smaug/dashboards
 $ kustomize edit add resource ${DASHBOARD_FOLDER}
 ```
 
+### Backing Up Your Dashboard:
+Grafana is redeployed frequently, such as during an automatic operator update or when a new dashboard is added via a pull request. Please make sure to backup any dashboards you have created through the Grafana UI.
+
+To export your dashboard click the share icon
+<img src="https://img.icons8.com/ios-glyphs/344/share--v1.png" width="25" height="25">
```

Comment:
Let's move the image into this repository (otherwise if the remote image disappears we end up with a broken image).